index.js.flow

```diff
@@ -58,3 +58,9 @@ declare export function launchCamera(callback: (response: Response) => any): voi
 
 declare export function launchImageLibrary(options: ?Options, callback: (response: Response) => any): void;
 declare export function launchImageLibrary(callback: (response: Response) => any): void;
+
+declare export default {
+  showImagePicker(options: ?Options, callback: (response: Response) => any): void
+  launchCamera(options: ?Options, callback: (response: Response) => any): void,
+  launchImageLibrary(callback: (response: Response) => any): void,
```

Comment:
@benjaminreid How come `launchImageLibrary` in the default export is missing the options param, is this intentional? This prevents flow from passing:

```
Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ 

Cannot call ImagePicker.launchImageLibrary because no more than 1 argument is expected by function type [1].

     src/xxx/xxx/index.js
     102│   handlePickImage = () => {
     103│     ImagePicker.launchImageLibrary({}, (response) => {
     104│       if (response.error) {
     105│         Toasts.raiseError(response.error);
     106│       } else if (!response.didCancel) {
     107│         this.setState({base64Image: response.data});
     108│       }
     109│     });
     110│   };

     node_modules/react-native-image-picker/index.js.flow
 [1]  65│   launchImageLibrary(callback: (response: Response) => any): void,

Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ 

Cannot call ImagePicker.launchImageLibrary with object literal bound to callback because a callable signature is missing
in object literal [1] but exists in function type [2].

     src/xxx/xxx/index.js
     102│   handlePickImage = () => {
 [1] 103│     ImagePicker.launchImageLibrary({}, (response) => {
     104│       if (response.error) {
     105│         Toasts.raiseError(response.error);
     106│       } else if (!response.didCancel) {

     node_modules/react-native-image-picker/index.js.flow
 [2]  65│   launchImageLibrary(callback: (response: Response) => any): void,
```

As you can see I don't actually have any options; I'm passing in an empty object. However, omitting the options param throws the following error:

<img width="416" alt="screen shot 2018-10-13 at 10 41 12 am" src="https://user-images.githubusercontent.com/646278/46898364-b92e2d80-ced4-11e8-8d53-67947b5cb6ce.png">